plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -342,7 +342,7 @@ func ClusterRoles() []rbac.ClusterRole {
 				// this is for leaderlease access
 				// TODO: scope this to the kube-system namespace
 				rbac.NewRule("create").Groups(legacyGroup).Resources("endpoints").RuleOrDie(),
-				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-scheduler").RuleOrDie(),
+				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-default-scheduler").RuleOrDie(),
```

Comment:
this just seems like change with little benefit... running schedulers with different names still won't work. it would make more sense to me to default to kube-system/kube-scheduler and put the namespace and name under the user's control